hs_discover/views.py

```diff
@@ -60,8 +60,13 @@ def get(self, request, *args, **kwargs):
         sort = 'modified'
         if request.GET.get('sort'):
             sort = request.GET.get('sort')
+            # protect against ludicrous sort orders
+            if sort != 'title' and sort != 'author' and sort != 'modified' and sort != 'created': 
+                sort = 'modified'
```

Comment:
I think we will want to know if a ludicrous sort order is requested.  Maybe return a bad request with the message of an unsupported sort order?

Reply:
This will **always** be a hacking attempt. They try things like that looking for vulnerabilities.

Reply:
I don't know how to return an error. This is an AJAX callback routine. THere's no provision in the AJAX output to return an error. This is too much work for now.